src/FuncHW/Program.cs

```diff
@@ -27,11 +27,18 @@ static void Main()
 
             //Func*** funcPersonIsActive = ***;
             //Func*** funcPersonPhoneNumberStartsWith7 = ***;
+            Func<Person, bool> funcPersonPhoneNumberStartsWith7 = (Person person) => 
+                person.PhoneNumber / 10000 == 7;
+
             //Func*** funcPersonIsChild = ***;
             //Func*** funcPersonHasShortName = ***;
 
             //Func*** funcCatIsDomestic = ***;
             //Func*** funcCatColorIsDark = ***;
+            Func<Cat, bool> funcCatColorIsDark=(Cat cat)=> 
```

Comment:
Be careful with spaces. `...Dark = (Cat cat) =>`

Reply:
Done